README.md

```diff
@@ -1,51 +1,62 @@
 # Resinhup
 Tool for resin host OS updates.
 
-## Instalation
-
-### Pull from resin registry
-```sh
-[host] $ rce pull registry.resinstaging.io/resinhup/resinhup-<<machine>>
-[host] $ rce tag -f registry.resinstaging.io/resinhup/resinhup-<<machine>> resinhup
-```
-Latest tag will point to the image build from the latest git tag. Replace &lt;&lt;machine&gt;&gt; by your target machine. Example raspberry-pi2. See https://github.com/resin-io/resinhup/blob/master/conf/resinhup for supported machines.
-
-### Local built rce/docker images
-Clone this repository on target (or transfer), change your current directoy in the one where you have the cloned copy of the repository and:
-```sh
-[host] $ cp Dockerfile.<machine> Dockerfile
-[host] $ rce build -t resinhup .
-```
-Replace &lt;&lt;machine&gt;&gt; by your target machine. Example raspberry-pi2. See https://github.com/resin-io/resinhup/blob/master/conf/resinhup for supported machines.
+Be aware that in the current stage of development **this tool is not meant to be ran by itself but through a** [wrapper](https://github.com/resin-os/meta-resin/blob/master/meta-resin-common/recipes-support/resinhup/resinhup/run-resinhup.sh) developed in [meta-resin](https://github.com/resin-os/meta-resin). This wrapper takes care of all the prerequisites needed for this tool and adds support for resin supervisor update too. In this way, using that wrapper, a device can be updated completely (HostOS + Supervisor).
+
+As well the current development stage uses **docker images/containers to deploy/run** this tool. This is because when we first developed this tool the HostOS was not providing all the python prerequisites needed for it to successfully work. The long term plan would be to bring it completely in the Host OS including all the prerequisites. This is not completely decided because HostOS has hard requirements on rootfs size and we try to keep it as small as possible. So this docker container solution is kept for now even though it adds the overhead of downloading an image before being able to run the updater.
+
+## Docker containers versus git repository
+The releases for resinhup are marked in this git repository as git tags. As well the git repository includes Dockerfiles for each resinhup supported resin board. For example there is a Dockerfile called _Dockerfile.raspberrypi3_ which is the Dockerfile for creating a resinhup docker image for [Raspberry Pi 3](https://www.raspberrypi.org/blog/raspberry-pi-3-on-sale/) boards. We currently upload our resinhup docker images to resin registry (registry.resinstaging.io). We do this because old resin devices were using docker 1.4 which can't pull from registry v2 dockerhub.
+
+For each resinhup release (tag) there will be a set of docker images with the same tag uploaded to the above mentioned registry. The images have the name format: _resinhup-<board-slug>_. The full docker images URL format becomes _registry.resinstaging.io/resinhup/resinhup-<board slug>_.
```

Comment:
Can you format the image names as `code`? Otherwise the board-slug doesn't show up